tutorials/auto_inertia_calculation.md

```diff
@@ -0,0 +1,116 @@
+
+>**Note:** Use SDF Spec version 1.11 or greater to utilize the new tags and attributes of this feature.
+
+Here's an example snippet of a cylinder model that has automatic inertial calculations enabled and has a density of 1240 kg/m^3:
+
+```xml
+  <model name="cylinder">
+    <link name="cylinder_link">
+      <inertial auto="true" />
+      <collision name="collision">
+        <density>1240.0</density>
+        <geometry>
+          <cylinder>
+            <radius>1</radius>
+            <length>2</length>
+          </cylinder>
+        </geometry>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <cylinder>
+            <radius>1</radius>
+            <length>2</length>
+          </cylinder>
+        </geometry>
+        <material>
+          <diffuse>1.0 1.0 0.0 1.0</diffuse>
+          <ambient>1.0 1.0 0.0 1.0</ambient>
+          <specular>1.0 1.0 0.0 1.0</specular>
+        </material>
+      </visual>
+    </link>
+  </model>
+```
+
+If you use the above snippet in an empty world and launch it with `gz-sim`, here's how it would look:
+
+![Cylinder](files/auto_inertia/cylinder_inertia_demo.gif)
+
+## Links with Multiple Collisions & the Effect of Density
+
+To understand the inertia calculation in links with multiple collisions and the effect of setting different density values, you can launch the `auto_inertia_pendulum.sdf` example world using:
+
+```bash
+gz sim auto_inertia_pendulum.sdf
+```
+
+After the gz-sim window opens up, you can right click on both the models and enable the centre of mass visualization by selecting the `View > Center of Mass` option from the menu. Once you play the simulation it should look this:
+
+![Pendulum](files/auto_inertia/auto_inertia_pendulum.gif)
+
+This example world has two structurally indentical models. The pendulum link of both the models contain 3 cylindrical collision geometries: One on the top which forms the joint, One in longer cylinder in middle and One at the end which forms the bob of cylinder. Even, though they are indentical, the center of mass for both are different as they use different density values for the different cylinder collisions. One one hand, the upper joint collision of the pendulum on the left has the highest density which causes the center of mass to shift closer to the axis while on the other hand, the bob collision of the pendulum on the right has the highest density which causes the center of mass to shift towards the end of the pendulum.
+This difference in mass distribution about the axis of rotation results in difference in the moment of inertia of the 2 setups and hence different angular velocities.
+
+## Mesh Inertia Calculation with Rolling Shapes Demo
+
+Let's try another example world, `auto_inertia_rolling_shapes.sdf`. This can be launched with `gz sim` using the following command:
+
+```xml

```

Comment:
```suggestion
```bash

```

Reply:
Corrected in [35df259](https://github.com/gazebosim/gz-sim/pull/2119/commits/35df259fd9c212f2605cabf03ba15dd07cc5ea49)